rke/k8s_version_info.go

```diff
@@ -643,6 +643,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.7-patch0",
 			MinRKEVersion:     "1.3.13-rc0",
 		},
+		"v1.25": {
+			MinRancherVersion: "2.7.1-patch0",
```

Comment:
@jiaqiluo would this be `2.7.2-patch0` if we're enabling with v2.7.2?

Reply:
discussed offline, the comment `// Enabled in Rancher v2.7.2` needs to be v2.7.1.

Reply:
fixed